From 495a27daa8ca91bb357a065c986552c3375eda82 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@redhat.com>
Date: Fri, 24 May 2019 18:40:01 +0100
Subject: [PATCH 1/3] x86_64-rh-devices: add missing TPM passthrough
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Marc-André Lureau <marcandre.lureau@redhat.com>
Message-id: <20190524184002.14050-2-marcandre.lureau@redhat.com>
Patchwork-id: 88230
O-Subject: [RHEL-AV-8.1.0 qemu-kvm PATCH 1/2] x86_64-rh-devices: add missing TPM passthrough
Bugzilla: 1519013
RH-Acked-by: Philippe Mathieu-Daudé <philmd@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert <dgilbert@redhat.com>

The TPM passthrough support got lost with 4.0 rebase, due to
configure/Kconfig changes.

Signed-off-by: Marc-André Lureau <marcandre.lureau@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 default-configs/x86_64-rh-devices.mak | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/default-configs/x86_64-rh-devices.mak b/default-configs/x86_64-rh-devices.mak
index 05ad6cf..8079fa7 100644
--- a/default-configs/x86_64-rh-devices.mak
+++ b/default-configs/x86_64-rh-devices.mak
@@ -92,3 +92,6 @@ CONFIG_VTD=y
 CONFIG_WDT_IB6300ESB=y
 CONFIG_WDT_IB700=y
 CONFIG_XIO3130=y
+CONFIG_TPM_CRB=y
+CONFIG_TPM_TIS=y
+CONFIG_TPM_PASSTHROUGH=y
-- 
1.8.3.1

